src/app/charts/catalogservice/values.yaml

```diff
@@ -15,7 +16,7 @@ scale: # Horizontal Pod Autoscaler (HPA) configuration
 
 ingress:
   annotations:
-    cert-manager.io/cluster-issuer: "cert-manager-alwayson"
+    cert-manager.io/cluster-issuer: "cert-manager-alwayson" # this will trigger cert-manager to request a certificate
```

Comment:
I'm thinking we should move this to the healthservice instead:
- we'll expect many customers might reuse our healthservice anyway. Then they dont need to move this part around for their own workload and decide on which one to annotate
- healthservice currently gets installed first anyway

Reply:
Yes, I agree. That makes sense.

Reply:
ok done